src/client.rs

```diff
@@ -450,7 +448,27 @@ impl Client {
     /// # Ok(()) }
     /// ```
     pub fn patch(&self, uri: impl AsRef<str>) -> RequestBuilder {
-        let uri = uri.as_ref().parse().unwrap();
-        RequestBuilder::new(Method::Patch, uri).with_client(self.clone())
+        RequestBuilder::new(Method::Patch, self.url(uri)).with_client(self.clone())
+    }
+
+    /// sets the base path for this client
```

Comment:
for consistent doc looks:
```suggestion
    /// Sets the base URL for this client. All request URLs will be relative to this path.
    ///
    /// # Examples
```